common/src/main/java/org/apache/uniffle/common/ShuffleDataResult.java

```diff
@@ -71,6 +71,10 @@ public byte[] getData() {
     return ByteBufUtils.readBytes(buffer.byteBuf());
   }
 
+  public int getDataLength() {
+    return getData().length;
```

Comment:
If we use off memory, we use the method `getData`, we will copy extra memory copy.

Reply:
good point!
let me think about it..

Reply:
I directly use `buffer` length instead of directly `getdata()`